### PR TITLE
Fix win32 tests

### LINKED ISSUE #1: Tests failing on Windows 32
I've disabled that build for now.

### PR DESCRIPTION
**Pull request by morris25**
Fixes #1

### CODE REVIEW

File: test/lz4frame.jl [hunk 1]
```diff
@@ -5,14 +5,15 @@
...
         @test LZ4.LZ4F_getErrorName(no_error) == "Unspecified error code"
-
-        @test LZ4.LZ4F_isError(ERROR_GENERIC)
-        @test LZ4.LZ4F_getErrorName(ERROR_GENERIC) == "ERROR_GENERIC"
+        
+        if Sys.WORD_SIZE == 64
+            ERROR_GENERIC = UInt(18446744073709551615)
+            @test LZ4.LZ4F_isError(ERROR_GENERIC)
+            @test LZ4.LZ4F_getErrorName(ERROR_GENERIC) == "ERROR_GENERIC"
+        end
```
Comment (omus) [inline test/lz4frame.jl]: Looking at this again should you just make `ERROR_GENERIC` a `UInt64`?

Comment (morris25) [reply test/lz4frame.jl]: ...probably.

Comment (morris25) [reply test/lz4frame.jl]: On second thought probably not. The C code to define the error is using a `ptrdiff_t` into a list of error codes, the value of ERROR_GENERIC should be completely different.

Comment (omus) [reply test/lz4frame.jl]: Ok, I think the standard way to check for this is `Int == Int64` as `Int` will change depending on if the system is 32-bit or 64-bit.

Comment (morris25) [reply test/lz4frame.jl]: Good to know. Thanks!

**Pull request merged**